### PR TITLE
Add API dashboards

### PR DESCRIPTION
**Pull request by keithf4**
Also fixed broken jobname dropdown in PostgreSQLDetails dashboard and made the datasource variable name consistent in the PostgreSQL dashboard.

### CODE REVIEW

**Comment by sharmay** [comment]
since curl commands can directly use original json why to add _API.json ?
I suggest do not add them to remove any confusion.

**Comment by keithf4** [comment]
Done. Tested all the dashboards with the command I added to the grafana README.

File: grafana/README.md [hunk 1]
```diff
@@ -8,8 +8,27 @@ The steps to access the customized dashboards are as follows:
...
   * PostgreSQL.json
   * PostgreSQLDetails.json
   * BloatDetails.json
   * CRUD_Details.json
+  * TableSize_Details.json
+
+
+### API Import
+
+It is possible it import these graphs through the "import" HTTP API using the following curl command to add some required wrapper information to each json blob.
```
Comment (sharmay) [inline grafana/README.md]: typo "It is possible it import" -> "It is possible to import"

File: grafana/README.md [hunk 2]
```diff
@@ -8,8 +8,27 @@ The steps to access the customized dashboards are as follows:
...
+
+### API Import
+
+It is possible it import these graphs through the "import" HTTP API using the following curl command to add some required wrapper information to each json blob. 
+```
+curl --user admin:admin -S "http://localhost:3000/api/dashboards/import" -X POST -H 'Content-Type: application/json;charset=UTF-8' --data-binary "{  \"dashboard\" : $(cat PostgreSQL.json) , \"overwrite\":true, \"inputs\":[  {  \"name\":\"DS_PROMETHEUS\", \"type\":\"datasource\", \"pluginId\":\"prometheus\", \"value\":\"PROMETHEUS\" } ] }"
+```
+You will likely have to edit the following parts of the above command:
+
+ * Username
```
Comment (sharmay) [inline grafana/README.md]: I suggest " Replace admin:admin with user:password with access to rest api ex: grafana admin"

File: grafana/README.md [hunk 3]
```diff
@@ -8,8 +8,27 @@ The steps to access the customized dashboards are as follows:
...
+ * Password
```
Comment (sharmay) [inline grafana/README.md]: Also add that they may also have to change URL if importing to grafana is done remotely.

Comment (sharmay) [inline grafana/README.md]: Depending on what prometheus datasource is named they will have to replace value.
\"value\":\"PROMETHEUS\"

File: grafana/README.md [hunk 4]
```diff
@@ -8,8 +8,27 @@ The steps to access the customized dashboards are as follows:
...
+ * Hostname (and port if different from default)
```
Comment (sharmay) [inline grafana/README.md]: See previous url comment instead hostname I suggest using URL

File: grafana/README.md [hunk 5]
```diff
@@ -8,8 +8,27 @@ The steps to access the customized dashboards are as follows:
...
+ * The file to import goes in the $cat() parentheses. In the above example this is "PostgreSQL.json".
+ * The name of your grafana datasource is the final json "value". In the above example this is "PROMETHEUS".
```
Comment (sharmay) [inline grafana/README.md]: See above command, to provide more info on this.

File: grafana/README.md (hunk 3 above)
Comment (keithf4) [reply grafana/README.md]: I already mention this in the last bullet.

File: grafana/README.md (hunk 5 above)
Comment (keithf4) [reply grafana/README.md]: You're above comment doesn't provide anything more clear on this.

**Pull request merged**